### DISCUSSION TITLE
Load Balancing - Plugin and Server on Same System?

### DISCUSSION THREAD

**Discussion by Kylyx** [apohl79/audiogridder#402] [Q&A]
Hey All!

Just came across this app today and am pretty blown away. Installed both on my PC running Studio One and watched a Insert of 14 plugins that had S1's CPU meter at 85% get reduced to 3% when added to an instance of AG. The best part is seeing the load move off of one core of my i9-10850K and spread out over all of them!

Question is, are there any downsides to using the app this way? It seems to mostly be pushed as a remote solution.

Thx!
Mike

**Comment by apohl79** [comment]
There is more people using it as a "local" solution. But as this was not the intended use case, there is some UI limitations (only one plugin UI at the same time which is not really integrated into your DAW).  You should disable screen capturing in the server settings. In the future I will try to make it more usable in a local only setup.

**Reply by Kylyx** [reply]
Yes, please do! It is revolutionary to let DAW users escape from "single core hell" and be able to focus on creativity and not technical limitations. Thanks for your work on this! I've got a couple of other questions but I'll put them in a new discussion...

**Reply by Yorshoff** [reply]
Yes, indeed. On one machine it helps to use CPU more properly and a bit more user friendly Local mode would be an awesome thing!

**Reply by JohnBlumstein** [reply]
+1

**Comment by jaacMusic** [comment]
Hi Mike,

Most DAWs out there support plugin load balancing and or DSP multithreaded processing. You should look into your DAW settings for an option. I'm not saying to move away from AG, but I"m surprise your DAW is eating one core when most DAW can take advance of all CPU cores/threads. Just an observation.

**Reply by Fittkenny** [reply]
jaacMusic, you are not a Logic user are you? The single core overload is famous and notorius.

**Reply by Kylyx** [reply]
Thx jaacMusic. Unfortunately, Studio One doesn't seem to have this ability baked in so you can imagine the usefulness of AG in these cases. Which is your DAW of choice? You're one of the lucky ones I take it?

**Reply by AtmanActive** [reply]
I am using Reaper on windows and can confirm: full multithreading working as expected. If you are on windows, one way to keep an eye on multi-core load at all times would be [XMeters](https://entropy6.com/xmeters/)

**Reply by jaacMusic** [reply]
Your are right. I'm using a Cakewalk in Windows. It does allows you to manage his DSP are load balance, like  many threads you would like to use for DSP and buffer blocks, etc. Right now with all CPUs focusing on more cores and threads it doesn't make sense going with a software that doesn't take advantage of them. CPU frequency is no longer something CPU manufactures focus on.

**Reply by Kylyx** [reply]
Funny enough, I also use Cakewalk and know that it does do this natively. I came across this app in search of solutions for a friend that uses S1 on a Mac and is having CPU overloading issues. The S1 single core loading is a known thing there. Someone on the Presonus forum had suggested AG as a solution. We tried it in a local configuration on his Mac but we're having trouble getting it to run properly. Kept getting a loop where the server would keep restarting. So, I installed it on my PC and was able to reproduce the single core issue and then watch it disappear when using AG. Hoping Andreas can make it more "friendly" in a local environment as he was suggesting. Would really need a couple of tweaks to be super useful tho, like the ability to drag and drop plugins into an AG plug-in from another channel. It's also weird not being able to see which plugins are on a channel at a glance.

**Reply by Yorshoff** [reply]
Currently only Reaper is good enough for spreading CPU resources between channels, but for guys like me (who hate Reaper ) it's a little consolation. I'm working in Cubase mostly and it handles CPU not so great, so AudioGridder with a good Local mode would be a god bless.

**Reply by belovedmike** [reply]
Old thread but hoping someone may reply. Now that version 4.5 and upwards of Studio One supports multi threading of CPU for plugins, would it be fruitless installing AG on the same system or would I likely still get some performance improvement in running lots of intensive plugins like AA? That may sound like a silly question but I'm not familiar with multi threading and whether there is one way of doing this or whether AG does this more efficiently somehow
